cli/azd/Makefile

```diff
@@ -0,0 +1,20 @@
+PROTO_DIR=grpc/proto
```

Comment:
Please add notes to https://github.com/Azure/azure-dev/blob/main/cli/azd/CONTRIBUTING.md about how to run this file to generate files when needed